src/stdio-bridge/stdio-bridge.c

```diff
@@ -86,7 +90,21 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_bus_path = optarg;
                         break;
 
-                default:
+                case ARG_MACHINE: {
+                        char *e = NULL;
+
+                        e = bus_address_escape(optarg);
+                        if (!e)
+                                return log_oom();
+
+                        arg_bus_path = e;
```

Comment:
I think it is necessary to free the previous value. That is, when we specify multiple --machine=, then this causes memory leak.

Reply:
Maybe, using free_and_replace() is a good idea, isn't it?